adapters/appnexus.go

```diff
@@ -235,3 +237,42 @@ func NewAppNexusAdapter(config *HTTPAdapterConfig, externalURL string) *AppNexus
 		usersyncInfo: info,
 	}
 }
+
+func CollectImpData(oreq openrtb.BidRequest, req *pbs.PBSRequest, unit pbs.PBSAdUnit) string {
+	data := ""
+
+	//set 0 for web impression, and 1 for mobile impression
+	if oreq.Site != nil {
+		data += "plat:0"
+	} else {
+		data += "plat:1"
+	}
+
+	switch unit.MediaTypes[0] {
+	case pbs.MEDIA_TYPE_BANNER:
+		data += ";fmt:0"
+	case pbs.MEDIA_TYPE_VIDEO:
+		data += ";fmt:1"
+	default: //todo change to/add case for pbs.MEDIA_TYPE_NATIVE when it's implemented
+		data += ";fmt:2"
+	}
+
+	if req.SDK != nil {
+		switch req.SDK.Platform {
+		case "iOS":
+			data += ";sdk:0"
```

Comment:
I don't know much about this data format, or how it ultimately gets used by appnexus, but... is it wise to send over ints? Is it possible, or better, to send over strings, like "ios" or "android"?

I ask because I imagine myself looking at a log message. If it says `sdk:0`, I have to ask "what does 0 mean?" and then (probably) look it up in some docs. If it says `sdk:ios`, it's self-explanatory, and we won't even need to write docs for it.

The same goes for abbreviations like `fmt` and `plat`. They're not _as_ bad, but they're still more likely to confuse some new hires than `format` and `platform` would be.

But... again, I'm pretty ignorant of the final system here. If we're trying to save space, or appnexus has built some infrastructure that lets you store the semantics somewhere and convert them on display, then these ints might still make more sense.

Reply:
@mkendall07 I can change the data to a readable format like Dave mentioned above.  Please let me know which way you'd like the data to be presented.

Reply:
It's just to save space. It probably doesn't matter too much either way.

Reply:
ok... if appnexus doesn't have any major rule on it, readability should trump storage space.

I probably won't be the one consuming this data... but I've been on the wrong end of data formats with 0s and 1s before. More readable data would have made my life much easier.